### PR TITLE
migrate to new api

### LINKED ISSUE #46: Update variables names
The name for variables should be updated to match the API responses.
Suggestion: The variables could drop the `@SerializedName` annotation and could have the same name as in the API response

### PR DESCRIPTION
**Pull request by aniri**
will close https://github.com/code4romania/mon-vot-android-kotlin/issues/46

### CODE REVIEW

File: app/src/main/java/ro/code4/monitorizarevot/data/model/FormDetails.kt [hunk 1]
```diff
@@ -27,16 +28,24 @@ class FormDetails {
...
+        if (this === other) return true
+        if (javaClass != other?.javaClass) return false
+
+        other as FormDetails
 
+        if (id != other.id) return false
+        if (code != other.code) return false
+        if (formVersion != other.formVersion) return false
+
+        return true
```
Comment (AlexandraDaraban) [inline app/src/main/java/ro/code4/monitorizarevot/data/model/FormDetails.kt]: Simplify this in a `return id==other.id &&...`

File: app/src/main/java/ro/code4/monitorizarevot/data/model/Question.kt [hunk 2]
```diff
@@ -23,25 +22,22 @@ class Question {
...
+    //1  single choice, 2 raspuns cu text, 3 multiple choice si raspuns liber 0 multiple choice
     @Expose
-    @SerializedName("idTipIntrebare") //1  single choice, 2 raspuns cu text, 3 multiple choice si raspuns liber 0 multiple choice
     var questionType: Int = 0
 
     @Expose
-    @SerializedName("raspunsuriDisponibile")
     @Ignore
-    lateinit var answers: List<Answer>
+//    @Relation(parentColumn = "id", entityColumn = "question_id")
```
Comment (AlexandraDaraban) [inline app/src/main/java/ro/code4/monitorizarevot/data/model/Question.kt]: Remove comment

File: app/src/main/java/ro/code4/monitorizarevot/data/model/Section.kt [hunk 3]
```diff
@@ -5,52 +5,47 @@ import androidx.room.ForeignKey
...
-    @SerializedName("codSectiune")
     var code: String? = null
 
     @Expose
-    @SerializedName("descriere")
     var description: String? = null
 
     @Expose
-    @SerializedName("intrebari")
 //    @Relation(parentColumn = "id", entityColumn = "section_id")
```
Comment (AlexandraDaraban) [inline app/src/main/java/ro/code4/monitorizarevot/data/model/Section.kt]: Remove this comment

File: app/src/main/java/ro/code4/monitorizarevot/modules/Modules.kt [hunk 4]
```diff
@@ -63,7 +63,7 @@ val apiModule = module {
     single {
         val interceptor = HttpLoggingInterceptor()
         interceptor.level =
-            if (!DEBUG) HttpLoggingInterceptor.Level.BASIC else HttpLoggingInterceptor.Level.NONE
+            if (!DEBUG) HttpLoggingInterceptor.Level.BODY else HttpLoggingInterceptor.Level.BODY
```
Comment (AlexandraDaraban) [inline app/src/main/java/ro/code4/monitorizarevot/modules/Modules.kt]: Should be `NONE` if the app is not in debug mode

**Review by AlexandraDaraban — CHANGES_REQUESTED** [review]
Hey, @aniri , looks good but there are some minor issues.

**Comment by aniri** [comment]
@AlexandraDaraban I fixed the issues :smile_cat:

**Review by AlexandraDaraban — APPROVED** [review]
Looks good! 😼

**Pull request merged**